src/Wallet.purs

```diff
@@ -102,6 +106,10 @@ mkNamiWalletAff = do
           Deserialization.UnspentOuput.convertUnspentOutput
             <$> fromBytesEffect (unwrap bytes)
 
+  getBalance :: NamiConnection -> Aff (Maybe Value)
+  getBalance nami = fromNamiHexString _getNamiBalance nami >>=
+    (_ >>= unwrap >>> fromBytes) >>> pure
```

Comment:
`fromBytes` here should probably change to `fromBytesEffect` (haven't had time yet to refactor).